code/modules/lore/organizations.dm

```diff
@@ -103,9 +103,10 @@
 	headquarters = "Mars"
 	motto = ""
 
-	ship_prefixes = list("NSV" = "exploration", "NTV" = "hauling", "NDV" = "patrol", "NRV" = "emergency response")
+	ship_prefixes = list("NCV" = "transport", "NSV" = "exploration", "NTV" = "hauling", "NDV" = "patrol", "NRV" = "emergency response", "NAV" = "inspection", "NBV" = "trading", "NXV" = "flight testing", "NMV" = "mining")
 	// Note that the current station being used will be pruned from this list upon being instantiated
 	destination_names = list(
+		"NSS Cyberiad in Epsilon Eridani",
```

Comment:
No. The Cyberiad is not in the destination names for a reason, it makes people confused if they get a message saying "Lol delivering to the Cyberiad!" with no followup.

Reply:
`// Note that the current station being used will be pruned from this list upon being instantiated`
This comment is basically poking out your eyes.
Once again, this is for future multi-station setups.

Reply:
There is no reason to include it. Just leave it the station name only not the system it is in.